src/utils/itemFilterUtils.ts

```diff
@@ -65,19 +66,34 @@ export const buildAppliedFiltersTagSetData = (
   return filters
 }
 
+/**
+ * Returns a tuple with the new filter values and the field that the value was removed from.
+ * We need to be able to remove a filter without prior knowledge of the filter group that contains it
+ * So that we can clear individual filters with the TagSet clearing buttons.
+ */
 export const removeValueFromFilters = (
-  idToRemove: string,
+  id: string,
   appliedFilters: AppliedItemFilters
-): [values?: string[], field?: string] => {
-  let valuesAndField: [values?: string[], field?: string] = [null, null]
-  Object.keys(appliedFilters).forEach((field) => {
-    const filterValueIndex = appliedFilters[field].indexOf(idToRemove)
-    if (filterValueIndex >= 0) {
-      appliedFilters[field].splice(filterValueIndex, 1)
-      valuesAndField = [appliedFilters[field], field]
-    }
+) => {
+  // find the filter field that includes the value to remove
+  const field = Object.keys(appliedFilters).find((field) => {
+    return appliedFilters[field].includes(id)
   })
-  return valuesAndField
+  // get a copy of the filter values with the value removed
+  const newValues = appliedFilters[field].filter(
+    (filterValue: string) => filterValue !== id
+  )
+  return [newValues, field]
+}
+
+export const getSelectedCheckboxesFromAppliedFilters = (
+  appliedFilters: AppliedItemFilters
+): SelectedCheckboxes => {
+  return {
```

Comment:
I don't think every multiselect needs to know the selcted values for all the other multiselects. The docs are a little unclear, but they say
> The value is an object with a key of the MultiSelect component's id value set to an object with an "items" key. The "items" value is an array of the selected items. 

I think this method might be extraneous, and you can generate the `selectedItems` values dynamically per multiselect on line 156 in `ItemFilters.tsx`